xml/admin_monitoring_alerting.xml

```diff
@@ -366,36 +366,129 @@ placement:
-      Refer to the receiver manual at
-      <link xlink:href="https://github.com/SUSE/prometheus-webhook-snmp#global-configuration-file" />.
-      for more details about the configuration options.
+       Alternatively, connect the container to the host network by using
+       the command line argument <literal>--network=host</literal>.
      </para>
-    </example>
-  </sect1>-->
+<screen>
+args:
+    - "--network=host"
+</screen>
+     <para>
+      Additionally, the SNMP host needs to be configured when the
+      SNMP trap receiver is not installed on the same host as the container.
+      Use the container's network gateway to be able to receive SNMP traps
+      outside the container/host:
+     </para>
+<screen>
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>CONTAINER_GATEWAY</replaceable>"
+</screen>
+     <sect2 xml:id="configure-prometheus-webhook-snmp">
+       <title>Configure the prometheus-webhook-snmp service</title>
+       <para>
+         The container can be configured by environment variables or by
+         using a configuration file.
+       </para>
+       <para>
+         For the environment variables, use <literal>ARGS</literal> to set global options and <literal>RUN_ARGS</literal>
+         for the run command options. You need to adapt the service specification
+         the following way:
+       </para>
+<screen>
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>CONTAINER_GATEWAY</replaceable>"
+    - RUN_ARGS="--metrics --port=9101"
+</screen>
+       <para>
+         To use a configuration file, the service specification must be adapted the following way:
+       </para>
+<screen>
+files:
+    etc/prometheus-webhook-snmp.conf:
+        - "debug: True"
+        - "snmp_host: <replaceable>ADD_HOST_GATEWAY_HERE</replaceable>"
+        - "metrics: True"
+volume_mounts:
+    etc/prometheus-webhook-snmp.conf: /etc/prometheus-webhook-snmp.conf
+</screen>
+       <para>
+         To deploy, run the following command:
+       </para>
+<screen>&prompt.cephuser;ceph orch apply -i <replaceable>SERVICE_SPEC_FILE</replaceable></screen>
+       <para>
+         See <xref linkend="deploy-cephadm-day2-services"/> for more
+         information.
+       </para>
+     </sect2>
+     <sect2 xml:id="configure-prometheus-alertmanager-for-snmp">
+       <title>Configure the &prometheus; &alertmanager; for SNMP</title>
+       <para>
+         Finally, the &prometheus; &alertmanager; needs to be configured specifically
+         for SNMP traps. If this service has not been deployed already, create
+         a service specification file. You need to replace IP_OR_FQDN with the IP
+         address or FQDN of the host where the &prometheus; &alertmanager; SNMP
+         webhook has been installed. For example:
+       </para>
+       <note>
+         <para>
+           If you have already deployed this service, re-deploy with the following
+           settings to ensure the &alertmanager; is set up correctly for SNMP.
```

Comment:
```suggestion
           If you have already deployed this service, then to ensure the
           &alertmanager; is set up correctly for SNMP, re-deploy with
           the following settings.
```

Reply:
Again, I'm trying to move _why_ to do something to before _what_ they have to do. I think this makes for easier reading comprehension. When I read things in the form "to get to Y from X, go this way" I stumble; I always find "from X to Y" easier.